Tests/Unit/DSL/Query/Span/SpanFirstQueryTest.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+/*
+ * This file is part of the ONGR package.
+ *
+ * (c) NFQ Technologies UAB <info@nfq.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ONGR\ElasticsearchBundle\Tests\Unit\DSL\Query\Span;
+
+use ONGR\ElasticsearchBundle\DSL\Query\Span\SpanFirstQuery;
+use ONGR\ElasticsearchBundle\DSL\Query\Span\SpanQueryInterface;
+
+/**
+ * Unit test for SpanFirstQuery.
+ */
+class SpanFirstQueryTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @var Object.
+     */
+    protected $mock;
+
+    /**
+     * Create mock object.
+     */
+    protected function setUp()
+    {
+        $this->mock = $this->getMockBuilder('ONGR\ElasticsearchBundle\DSL\Query\Span\SpanQueryInterface')->getMock();
+        $this->mock->expects($this->atMost(1))
+            ->method('getType')
+            ->will($this->returnValue('span_or'));
+        $this->mock->expects($this->atMost(1))
+            ->method('toArray')
+            ->will($this->returnValue(['key' => 'value']));
+    }
+
+    /**
+     * Reset mock object.
+     */
+    public function tearDown()
+    {
+        $this->mock = null;
```

Comment:
If you want to save memory use `unset()` because `= null` does not remove variable from symbol table.